plugin/src/automation/lift.ml

```diff
@@ -671,20 +670,96 @@ let do_lift_defn env evd (l : lifting) def =
 (*                           Inductive types                            *)
 (************************************************************************)
 
-let define_lifted_eliminator ?(suffix="_sigT") ind0 ind sort =
-  let env = Global.env () in
-  let ident =
-    let ind_name = (Inductive.lookup_mind_specif env ind |> snd).mind_typename in
-    let raw_ident = Indrec.make_elimination_ident ind_name sort in
-    Nameops.add_suffix raw_ident suffix
+let with_suffix n s =
```

Comment:
Yeah, I wanted to reuse a library function, but this situation has to manipulate values of `Name.t` instead of `Id.t`, annoyingly. I pushed a commit to inline the only two uses of that function, to avoid aliasing a library function at a different type. Alternatively, I could create the needed function, perhaps called `with_suffix_maybe` to disambiguate, in the plugin library?